lib/erb_lint/runner_config.rb

```diff
@@ -45,24 +45,30 @@ def merge!(other_config)
     end
 
     class << self
-      def default
+      def default(default_enabled: true)
+        default_enabled ||= default_enabled.nil? ? true : default_enabled
```

Comment:
nit: when we were chatting about this, I was thinking about something like this:

```ruby
foo ||= true
```

For a non-boolean input, that'll work fine, but we have to do the ternary because it's a boolean. Given that, the `||=` is not necessary and you can just use `default_enabled =` 🙂